plugins/reporters/asciidoc/src/funTest/kotlin/ManPageTemplateReporterFunTest.kt

```diff
@@ -33,13 +33,16 @@ import org.ossreviewtoolkit.utils.test.matchExpectedResult
 class ManPageTemplateReporterFunTest : StringSpec({
     "ManPage report is created from default template" {
         val expectedResultFile = getAssetFile("manpage-template-reporter-expected-result.1")
+        val reporter = ManPageTemplateReporter()
```

Comment:
Nit: To align with `HtmlTemplateReporterFunTest`, the empty line should be before this line, not after it, to group the `reporter` assigment to `generateReport`.

Reply:
Sorry, saw too late that the PR checks had already passed, and I couldn't disable auto-merge anymore to give you the choice to address the nit or not.